playbooks/roles/airship-deploy-tempest/templates/tempest.yaml.j2

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
why aren't this chart yaml in the soc site directory?

Reply:
I'll move this chart there.